admin/includes/classes/AdminRequestSanitizer.php

```diff
@@ -411,15 +426,18 @@ private function filterProductDescRegex($parameterName)
     {
         $prodDescRegex = '~(load=|= ?\(|<![^-])~i';
         if (isset($_POST[$parameterName])) {
+            // Add the parameterName to the base arrayname.
+            $this->arrayName = $this->setCurrentArrayName($parameterName, $this->arrayName);
             $this->debugMessages[] = 'PROCESSING PRODUCT_DESC_REGEX == ' . $parameterName;
             if (is_array($_POST[$parameterName])) {
                 foreach ($_POST[$parameterName] as $pKey => $pValue) {
+                    $currentArrayName = $this->arrayName . '[' . $pKey . ']';
```

Comment:
Argh, an instance of needing to substitute the function to build `currentArrayName`.  Amazing what is discovered when changes are colorized.